edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ztp-image-based-upgrade-for-sno"]
+= Performing an image-based upgrade for {sno} clusters using GitOps ZTP
+:context: ztp-image-based-upgrade
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+You can upgrade your managed {sno} cluster with the image-based upgrade through GitOps ZTP.
+
+When you deploy the {lcao} on a cluster, an `ImageBasedUpgrade` CR is automatically created.
+You update this CR to specify the image repository of the seed image and to move through the different stages.
+
+// Lifecycle Agent (LCAO)
+
+include::modules/ztp-image-based-upgrade-talm-prep.adoc[leveloffset=+1]
+
+////
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/ztp-preparing-the-hub-cluster.adoc#ztp-preparing-the-ztp-git-repository-ver-ind_ztp-preparing-the-hub-cluster[Preparing the GitOps ZTP site configuration repository for version independence]
+
+* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc[Preparing for the image-based upgrade for single-node OpenShift clusters]
+
+* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-image.adoc#ztp-image-based-upgrade-shared-container-directory_shared-container-directory[Configuring a shared container directory between ostree stateroots when using GitOps ZTP]
+
+* xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#oadp-about-backup-snapshot-locations_installing-oadp-ocs[About backup and snapshot locations and their secrets]
+
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc[Creating a Backup CR]
+
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[Creating a Restore CR]
+////
+
+include::modules/ztp-image-based-upgrade-talm-upgrade.adoc[leveloffset=+1]
+
+include::modules/ztp-image-based-upgrade-talm-rollback.adoc[leveloffset=+1]
+
+include::modules/cnf-image-based-upgrade-troubleshooting.adoc[leveloffset=+1]
```

Comment:
This one is not getting pulled into the assembly :thinking: 
![image](https://github.com/openshift/openshift-docs/assets/61474374/3447721f-51b7-4aa2-87b4-c8b4824cd4cb)

Reply:
nvm this shows up now idk what the deal was